src/app/[lang]/about/page.tsx

```diff
@@ -1,29 +1,66 @@
 "use client";
+
+import AboutSection from "@/components/about-section";
 import AboutSectionLayout from "@/components/about-section-layout";
+import AppLink from "@/components/link";
 import ProjectCard from "@/components/project-card";
 import { useDictionary } from "@/contexts/dictionary-provider";
 
 export default function About() {
   const dict = useDictionary();
+
   return (
-    <main>
-      <AboutSectionLayout
-        linkName="see_more"
-        titleOrientation="vertical"
-        title={dict.about.projects.title}
-        subtitle={dict.about.projects.description}
-        href="/projects"
-        overflows
-      >
-        <div
-          className="flex flex-col md:flex-row md:gap-4"
-          style={{ scrollbarWidth: "none", msOverflowStyle: "none" }}
-        >
-          <ProjectCard type="sei" />
-          <ProjectCard type="bugsbyte" />
-          <ProjectCard type="coderdojo" />
+    <>
+      <div className="flex flex-col">
+        <div className="flex flex-col bg-white">
+          <AboutSectionLayout
+            linkName="see_more"
+            titleOrientation="vertical"
+            title={dict.about.projects.title}
+            subtitle={dict.about.projects.description}
+            href="/projects"
+          >
+            <div
+              className="flex flex-col md:flex-row md:gap-4"
+              style={{ scrollbarWidth: "none", msOverflowStyle: "none" }}
+            >
+              <ProjectCard type="sei" />
+              <ProjectCard type="bugsbyte" />
+              <ProjectCard type="coderdojo" />
+            </div>
+          </AboutSectionLayout>
         </div>
-      </AboutSectionLayout>
-    </main>
+        <AboutSection>
+          <div className="border-t border-black/10 pt-10">
+            <h2 className="w-full font-title text-3xl font-medium md:mb-4 md:mt-8 md:text-4xl">
```

Comment:
This title is waaay to big on desktop, according to Figma it should be 30px -> "text-3xl" on desktop and 24px on mobile -> "text-2xl".

Removed the margins because of what I explained in the other comment.

```suggestion
            <h2 className="w-full font-title text-2xl md:text-3xl font-medium">
```

I used the "sm:" breakpoint instead of "md:" to maintain a uniform design, because it's what AboutSectionLayout uses.